package ua.charity.GetHelpUaBot.exceptions;

public class CallbackNotFoundException  extends RuntimeException {
    public CallbackNotFoundException(String message) {
        super(message);
    }
}
